template/lib/utils.bash

```diff
@@ -42,10 +80,14 @@ download_release() {
   filename="$2"
 
   # TODO: Adapt the release URL convention for <YOUR TOOL>
-  url="$GH_REPO/archive/v${version}.tar.gz"
-
+  if [ $IS_GITHUB -eq 0 ]; then
+    url="$REPO/archive/v${version}.tar.gz"
```

Comment:
this should be `$REPO/releases/

Reply:
I've found an interesting difference in these endpoints, now that I look at it:

```
# using ccache/ccache as example repo
❯ curl -LI -w "code: %{response_code}\ncontent-type: %{content_type}\neffective_url: %{url_effective}\ntime_total: %{time_total}\n" https://github.com/ccache/ccache/archive/v4.7.4.tar.gz
...
code: 200
content-type: application/x-gzip
effective_url: https://codeload.github.com/ccache/ccache/tar.gz/refs/tags/v4.7.3
time_total: 0.158385

❯ curl -LI -w "code: %{response_code}\ncontent-type: %{content_type}\neffective_url: %{url_effective}\ntime_total: %{time_total}\n" https://github.com/ccache/ccache/releases/download/v4.7.4/ccache-4.7.4.tar.gz
...
code: 200
content-type: application/octet-stream
effective_url: https://objects.githubusercontent.com/github-production-release-asset-2e65be/461102/81edd037-87b4-4b75-ad06-9325550c3815?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20230122%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230122T152917Z&X-Amz-Expires=300&X-Amz-Signature=fe99692a02cfdca69367c08a6205aec3a74b2736d45e4cc55bd7bd450547c9ae&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=461102&response-content-disposition=attachment%3B%20filename%3Dccache-4.7.4.tar.gz&response-content-type=application%2Foctet-stream
time_total: 0.366820
```

The former appears to be a direct access link, and the latter is going to S3 (and has Varnish in front of it, as it also has a `via: 1.1 Varnish` response header; it was also a cache miss).

Both include `x-content-type-options: nosniff` and `content-disposition: attachment; filename=ccache-4.7.4.tar.gz`, but the former is explicitly declaring the type to be `gzip`, and the latter directs the browser to figure it out.

Either way, I don't care and can adjust if needed, but wanted to clarify that both are valid.